src/Raven.Server/Web/Studio/DataDirectoryInfo.cs

```diff
@@ -230,8 +230,8 @@ public GetDataDirectoryInfoCommand(string path, string name, bool isBackup)
             public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
             {
                 var path = _isBackup
-                    ? $"databases/{_name}/admin/backup-data-directory?path={_path}"
-                    : $"admin/studio-tasks/full-data-directory?path={_path}&name={_name}";
+                    ? $"databases/{_name}/admin/backup-data-directory?path={Uri.EscapeDataString(_path)}"
```

Comment:
check for null values!
in /admin/studio-tasks/full-data-directory path isn't required.